src/components/Overlay.ts

```diff
@@ -1,12 +1,12 @@
-import { DOM, SFC, SyntheticEvent } from "react";
```

Comment:
There is already a transparent later from web modeler. Delete `Overlay.ts`. 
Please confirm this from the web modeler